src/libs/KeyboardShortcut/index.js

```diff
@@ -60,13 +61,28 @@ function getDisplayName(key, modifiers) {
     return displayName.join(' + ');
 }
 
+/**
+ * Check if the Enter key was pressed during IME confirmation (i.e. while the text is being composed).
+ * See {@link https://en.wikipedia.org/wiki/Input_method}
+ * @param {Event} event
+ * @returns {boolean}
+ */
+const isEnterWhileComposition = (event) => {
```

Comment:
The naming for this method is a little confusing, what is the difference between this and when the enter key is pressed?

Reply:
Usually, when you type in English, the moment you inputted a character, the character is considered already "composed". However, in other languages that uses IME, the character you type is not considered "composed", meaning that the text is still in "draft" mode. When you type `Enter` while the text is still draft mode, it's `enterWhileComposing`. However, once the text is confirmed ("composed"), it's `Enter` key event that happens outside composition flow.

I took hint for the variable name from `KeyboardEvent.isComposing` ([doc](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/isComposing)).